components/scripts/gradle/gradle-init-scripts/configure-build-validation.gradle

```diff
@@ -54,6 +54,8 @@ def registerBuildScanActions = { def buildScan, def rootProjectName ->
     buildScan.onError { error ->
         def errorFile = new File(expDir, 'errors.txt')
         errorFile.text = 'Build Scan publishing failed.'
+        def exceptionFile = new File(expDir, 'exceptions.txt')
+        exceptionFile.text = error
```

Comment:
It took me a moment to look up what `error` is.  It looks like it's a Strong containing a JSON object that describes the error. So I think writing out the error variable like this is sufficient.